bin/entrypoint.sh

```diff
@@ -0,0 +1,219 @@
+#!/bin/bash
+
+usage() {
+cat << EOF
+usage: $0 [<options>]
+       $0 <command> [<options>]
+
+Available commands:
+  clean                        delete all files and directories generated by prepare
+  help                         show this help
+  prepare                      only prepare repository (do not serve or publish)
+  publish                      generate the site (into /public by default)
+  server                       serve your current site (default to localhost:1313)
+
+Flags:
+  -p, --port int               port on which the server will listen (default 1313)
+  -D, --[no]buildDrafts        include (or exclude) content marked as draft (default is including for server and excluding for publish)
+      --[no]prepare            include (or exclude) the preparation phase ie. retrieve modules and sub repositories (default prepare)
+EOF
+}
+
+# Default values for parameters
+
+PORT=1313
+ENVIRONMENT=development
+COMMAND="server"
+WITHDRAFTS=2
+BASEURL=""
+PREPARE=true
+
+FRONTLINE_VERSIONS=("1.13" "1.14")
+FRONTLINE_VERSION_LATEST="1.14"
+
+dir=$(cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")" && pwd -P)
+
+cd "$dir/.."
+
+MYTMPDIR="$(mktemp -d)"
+
+if [ -z "$DEBUG" ]; then
+  trap 'rm -rf -- "$MYTMPDIR"' EXIT
+fi
+
+clean () {
+  echo "=====> cleaning phase"
+  rm -Rf \
+    ./content/* \
+    ./package.json \
+    ./package-lock.json \
+    ./go.sum \
+    ./node_modules \
+    ./resources 
+}
+
+build_indexes() {
+  local repo="$1"
+  local branch="$2"
+  local remote_dir="$3"
+  local local_dir="$4"
+
+  export REPOSITORY="$repo" BRANCH="$branch" REMOTE_DIR="$remote_dir" LOCAL_DIR="$local_dir"
+  envsubst '${REPOSITORY} ${BRANCH} ${REMOTE_DIR} ${LOCAL_DIR}' < "template/indexes/unversionned-section-index.md" > "content/$local_dir/_index.md"
```

Comment:
```suggestion
 REPOSITORY="$repo" \
   BRANCH="$branch" \
   REMOTE_DIR="$remote_dir" \
   LOCAL_DIR="$local_dir" \
   envsubst '${REPOSITORY} ${BRANCH} ${REMOTE_DIR} ${LOCAL_DIR}' < "template/indexes/unversionned-section-index.md" > "content/$local_dir/_index.md"
```

Shouldn't need to export